bson/src/test/unit/org/bson/json/JsonReaderTest.java

```diff
@@ -119,6 +132,16 @@ public void testDateTimeShell() {
     }
 
     @Test
+    public void testDateTimeShellNow() {
+        long currentTime = new Date().getTime();
+        String json = "ISODate()";
```

Comment:
Thanks @rstam - we would have missed the difference for `Date` if it wasn't for your eagle eyes!